noxfile.py

```diff
@@ -73,6 +73,20 @@ def get_example_files():
     return [folder + "/" + file for file in filtered_files]
 
 
+# We want to mimic
+# git ls-files --other examples
+def get_example_other_files():
```

Comment:
This feels extremely similar to `git clean -xd -- <folder>` (check `git clean -h`)

Reply:
I have no idea what that command is, so probably yes